docassemble/MotiontovacateDefaultJudgment/data/questions/Motion_to_Vacate.yml

```diff
@@ -70,35 +95,43 @@ code: |
  date_to_file = original_courtdate.plus(days=11)
 ---
 question: |
-  Have you received a notice from a sheriff/constable about being removed from your home or apartment?
+  Did you get a notice from a sheriff or a constable?
+subquestion: |
+  Did a sheriff or constable send you a notice that says they are going to "remove" or move you and your things out of your home on a certain date?
 yesno: sheriff_notice
 ---
 question: |
  What is the reason you missed your hearing?
 fields: 
   - no label: reason_for_missing
+    datatype: checkboxes
```

Comment:
If you adopt this change from select box/dropdown to checkboxes, which means you can choose more than one, you will may have to update your tests.